guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -302,21 +302,21 @@ $ bundle exec rake test
 
 #### For a Particular Component
 
-You can run tests only for a particular component (e.g. Action Pack). For example,
-to run Action Mailer tests:
+To test a particular component (e.g. Action Pack) every component has a dedicated
+`bin/test` script. For example, to run Action Mailer tests:
 
 ```bash
 $ cd actionmailer
-$ bundle exec rake test
```

Comment:
This is the right thing to run. I don't know why it wouldn't've been working for you. :confused:

Reply:
It did work! So if it is, then nothing's wrong :)

Reply:
It didn't work at first because I forgot `bundle exec` (Classic!)

When I added `bundle exec` it worked but by then @kaspth had mentioned `bin/test`.